static/js/containers/PostPage.js

```diff
@@ -183,10 +179,6 @@ class PostPage extends React.Component<*, void> {
         )
         moderators = response
       }
```

Comment:
we're no longer doing anything with `moderators` here, so I think we can just change this to be

```js
if (!moderators) {
  await dispatch(actions.channelModerators.get(channelName))
}
```

Reply:
Kinda odd that reassigning a `const` worked in the first place 🤔